app/entry.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+// webpack assets
```

Comment:
great job organizing your entry point to this app. It reads very well from top to bottom